layers/+lang/python/packages.el

```diff
@@ -110,7 +110,24 @@
 
 (defun python/init-hy-mode ()
   (use-package hy-mode
-    :defer t))
+    :defer t
+    :init
+    (progn
+      (add-hook 'hy-mode-hook 'paredit-mode)
```

Comment:
Paredit-mode is shown in the [hy-mode](https://github.com/hylang/hy-mode) repository. That was my only reason. I will change this to `smartparens-mode` and add it to the `python/post-init-smartparens` soon.